internal/api/tests/achievement_handler_test.go

```diff
@@ -30,25 +31,62 @@ func TestCreateAchievementHandler(t *testing.T) {
 		input              string
 		setup              func(mockSvc *mocks.Service)
 		expectedStatusCode int
+		expectedResponse   string
 	}{
 		{
```

Comment:
Please add fail test case for
- GetParamsByID
- GetUserIDFromContext

Reply:
Done